docs/Hosting on Glitch.md

```diff
@@ -0,0 +1,55 @@
+![Glitch Logo](https://miro.medium.com/max/541/1*qilnXz9CPIMPpr-eN6pNyg.png)
+# 🌥️ Hosting on Glitch.com
+The free hosting service - useful for people that don't want to pay for a host.
+
+Pros: It works surprisingly well and you shouldn't have any major issues if you follow the instructions in this guide correctly.
+
+Cons: You'll have to wipe old logs every now and then when your upper limit of about 200MB is capped. If you're okay with not having old logs, this is the host for you. Remember, its hard to fill up 200MB worth of text so you should be good for a while.
+
+## Prerequisites
+* Create a Glitch account at https://glitch.com/signin
+> I would recommend registering with your GitHub account. This will come in handy later.
+* Create an UptimeRobot account at https://uptimerobot.com/signUp
+> You will need this to keep your bot up and running 24/7.
+  
+## Bot Setup
+1. Once your account is created, go to this link https://glitch.com/~glitch-discord-modmail and click "View Source".
+2. Then, click the project name in the top left corner and hit "Remix Project".
+3. Now, go to the `.env` file enter your bot token where it says `TOKEN=`. Note there can be no spaces before or after your token in that blank since it is a shell file.
+4. Next, there is a file named `config.json`. There, you will follow the steps from [🛠️ Setting up the bot](docs/setup.md#single-server-setup) and [📝 Configuration](docs/configuration.md) to configure the configuration file to fit your needs.
```

Comment:
config.json is deprecated; config.ini should be used instead

Reply:
I prefer .json and it works just fine

Reply:
You preferring to use the deprecated syntax does not mean that you should continue to push the deprecated mode in new documentation.  This brings inconsistency and thereby potential confusion into the documentation.